docs/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  overflow-x: hidden;
+}
+.feature-title {
+  text-align: center;
+}
+.left-align {
+  text-align: left;
+}
+.right-align {
+  text-align: right;
+}
+.bg-color-orange {
+  background: rgb(237, 89, 47) !important;
+}
+.browser-info {
+  text-transform: capitalize;
+}
+.feature-content {
+  padding-left: 6em !important;
+  padding-right: 6em !important;
+}
+
+.responsive-img {
+  height: auto;
+  width: 100%;
+  max-width: 500px;
+  min-width: 150px;
+}
+.grid {
+  padding-left: 4em;
+  padding-right: 4em;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+  justify-items: start;
+  background: rgb(245, 245, 245);
+}
+.grid > div {
+  padding: 2em;
+  margin: 0.5em;
+}
+.grid2 {
+  padding-left: 2em;
+  padding-right: 2em;
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+  justify-items: end;
+}
+.grid2 > div {
+  padding: 2em;
+  margin: 0.5em;
+}
+
+hr {
+  width: 50%;
+  margin: auto;
+}
+
+.feature-table {
+  margin: auto;
+}
+@media all and (max-width: 600px) {
+  .logo {
+    margin-left: 3em !important;
+  }
+  .grid {
+    padding-left: 0em !important;
+    padding-right: 0em !important;
```

Comment:
Bulma provides zero padding and zero margin classes. Using them would be better than redefining them in your own CSS.

Reply:
@dhruvkb @makkoncept How can I can apply for this one as it will added only for small screen.
Should I use JavaScript?

Reply:
[CSS Media Queries on W3Schools](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)

Reply:
I have added the same thing, but you are suggesting me to use class.

Reply:
@dhruvkb

Reply:
Oh, my bad. Sorry. This is fine as it is.